### DISCUSSION TITLE
Looking for help troubleshooting

### DISCUSSION THREAD

**Discussion by swinggraphics** [stefansundin/bluesnooze#2] [Q&A] [answered]
A few weeks ago, I chose the option to hide the icon. How can I get it back? I don't see an obvious preference file to try deleting.

Bluesnooze stopped working recently, possibly at the first time I restarted since hiding the menu icon. I can see it running in Activity Monitor. How should I go about troubleshooting?

**Comment by stefansundin** [comment, marked as answer]
Hello. Sorry that I missed this discussion.

If you're running v1.3 or later then you can get the icon back by launching the application again. I don't remember if there's a way to do it on earlier versions.

I just released v1.4 and it explains how to do it every time you hide the icon. It also has some other new features like Wi-Fi support.

**Comment by swinggraphics** [comment]
I see what you mean now. Not launch the application again (I had quit and relaunched it several times, including when upgrading), but double click its icon when it's already running. Got it back! 👍
